### PR TITLE
Updates copy on /internet-of-things/appstore

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates copy on /internet-of-things/appstore

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things/appstore
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/1CKwmPDDtXOIgw-jSZFwzP-qLMc_rS_dUACpHQXlg0fA/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9927

### CODE REVIEW

**Comment by SirSamTumless** [comment]
@Hollzz Please could you review the demo above and give this ticket a PdM +1 label if you're happy with it.

**Comment by anasereijo** [comment]
- [ ] Holly is currently changing copy on the copydoc (fiy @SirSamTumless)
- [ ] call to action missing in "Securely manage your IoT fleet" section
- [ ] changes on "Ready to innovate?" section:
- [ ] hide image on mobile
Screenshot 2021-06-23 at 10 00 54

**Comment by Hollzz** [comment]
Looks good, I confirmed the correct changes in the copy doc, just a couple points need updating (which are commented in the copy doc):
- [ ] Green button and call to action in the "Securely manage your IoT fleet" section
- [ ] I updated "brand store" to be "app store" in the "What are the use cases?" section (3 occurrences)
- [ ] Green button and updated link in the "Enterprise-ready" section
- [ ] Added button and call to action in the "Dedicated App Store" section

Can I also make sure that the window/tab title will be "The IoT App Store for Linux" and not "White label app store for Linux and IoT". 

Thank you!

**Comment by SirSamTumless** [comment]
@petesfrenchI don't know if you saw the above.

@Hollzz We can do the copy changes to this doc this time. In the future, we'd have to push this out of the iteration, as copy needs to be final before we can tackle it.

**Review by Hollzz — APPROVED** [review]

**Comment by Hollzz** [comment]
I have spent a while trying to find out how to add a PdM +1 label and can't seem to... but I approved the changes and I'm happy with the demo.

**Comment by anthonydillon** [comment]
@Hollzz I've bumped your permissions so you should see the label editing now.

**Comment by anasereijo** [comment]
@petesfrench this is looking good thanks

you just missed a chevron from a link:

Screenshot 2021-06-29 at 14 02 42

**Comment by Hollzz** [comment]
@anthonydillon Thank you! Now I can :)

File: templates/internet-of-things/appstore.html [hunk 1]
```diff
@@ -52,95 +52,106 @@ <h2>Ready for business</h2>
...
-            height="43",
-            width="144",
+            width="131",
+            height="53",
             hi_def=True,
-            attrs={"class": "p-inline-images__logo"},
-            loading="lazy",
-          ) | safe
-        }}
+            loading="auto|lazy"
```
Comment (bethcollins92) [inline templates/internet-of-things/appstore.html:104]:
```suggestion
            loading="lazy"
```

File: templates/internet-of-things/appstore.html [hunk 2]
```diff
@@ -185,166 +196,134 @@ <h3 class="p-heading--four">System integrators</h3>
...
           </div>
         </div>
       </li>
       <li class="p-matrix__item">
         <div class="p-matrix__content">
-          <h3 class="p-heading--four">Usage analytics</h3>
+          <h3 class="p-heading--four">Line of products</h3>
           <div class="p-matrix__desc">
-            <p>Track the use of your devices over a dashboard. The brand store taps into the APIs provided by Ubuntu Core to extract usage metrics and present them through a user-friendly interface.</p>
+            <p>Brand stores can be dedicated to specific IoT device series in a model family. Each device model will have a store through which applications are published and managed.</p>
```
Comment (bethcollins92) [inline templates/internet-of-things/appstore.html:267]:
```suggestion
            <p>App stores can be dedicated to specific IoT device series in a model family. Each device model will have a store through which applications are published and managed.</p>
```

**Review by bethcollins92 — APPROVED** [review]
LGTM with a few suggestions inline

**Pull request merged**